package.json

```diff
@@ -21,31 +20,33 @@
     "url": "git+https://github.com/Microsoft/PowerBI-visuals-Tornado.git"
   },
   "devDependencies": {
-    "@types/jasmine": "2.5.37",
-    "@types/jasmine-jquery": "1.5.28",
-    "coveralls": "2.11.15",
-    "jasmine": "2.5.2",
+    "@types/d3": "3.5.36",
+    "@types/jasmine": "2.5.47",
+    "@types/jasmine-jquery": "1.5.30",
+    "@types/lodash": "4.14.55",
+    "coveralls": "2.13.1",
+    "jasmine": "2.6.0",
     "jasmine-jquery": "2.1.1",
-    "karma": "1.3.0",
-    "karma-chrome-launcher": "2.0.0",
+    "karma": "1.7.0",
+    "karma-chrome-launcher": "2.1.1",
     "karma-coverage": "1.1.1",
-    "karma-jasmine": "1.0.2",
-    "karma-remap-istanbul": "0.4.0",
+    "karma-jasmine": "1.1.0",
+    "karma-remap-istanbul": "0.6.0",
     "karma-sourcemap-loader": "0.3.7",
-    "karma-typescript-preprocessor": "0.3.0",
-    "lodash": "4.16.2",
-    "powerbi-visuals-tools": "1.4.3",
-    "powerbi-visuals-utils-chartutils": "^0.2.3",
-    "powerbi-visuals-utils-colorutils": "^0.2.1",
+    "karma-typescript-preprocessor": "0.3.1",
+    "lodash": "4.17.4",
+    "powerbi-visuals-tools": "1.6.4",
+    "powerbi-visuals-utils-chartutils": "0.3.0",
+    "powerbi-visuals-utils-colorutils": "0.2.1",
     "powerbi-visuals-utils-dataviewutils": "^1.0.0",
```

Comment:
Please remove the ```^``` sign. It would be good to have a persistent version of the package (I suppose that we should take a look at npm shrinkwrap. However, it isn't important now).